src/localStorage.js

```diff
@@ -0,0 +1,55 @@
+export default (() => {
+  const sortindex = (todoList) => todoList.map((item, index) => {
+    item.index = index;
+    return item;
+  });
+  // Store Tasks in Local Storage //
+  const storetask = (todoList) => {
+    localStorage.setItem('todo-list', JSON.stringify(todoList));
+  };
+
+  const getData = () => JSON.parse(localStorage.getItem('todo-list'));
+  // Tasks in the list
+  const add = (event) => {
+    const list = getData();
+    list.push({ description: event.target.value, completed: false, index: 0 });
+    storetask(sortindex(list));
+  };
```

Comment:
- [ ] The new tasks should have the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). in other words the index of the first item on the list suppose to equal `1` not `0` and the second item index equal `2` and so on. 

![image](https://user-images.githubusercontent.com/68796232/178011646-66a9391a-4d5d-4245-bab8-11a8febc56d9.png)